contracts/DaoConfiguration.sol

```diff
@@ -1,17 +1,17 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.0;
 
-import "./DaoBondCollateralWhitelist.sol";
+import "./DaoCollateralWhitelist.sol";
 
-abstract contract DaoBondConfiguration is DaoBondCollateralWhitelist {
-    struct DaoBondConfig {
+abstract contract DaoConfiguration is DaoCollateralWhitelist {
```

Comment:
should these two contracts go into a `contracts/dao` directory or are they ok out there in `contracts`?

Reply:
It may make sense to move them.

question: Is that refactoring best done as part of this PR or another?

Reply:
this refactoring is necessary i feel

Reply:
Yes, but to get this PR over the line, or after this PR?